backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/CalendarEventService.cs

```diff
@@ -0,0 +1,34 @@
+﻿using AutoMapper;
+using EasyMeets.Core.BLL.Interfaces;
+using EasyMeets.Core.Common.DTO.Calendar;
+using EasyMeets.Core.DAL.Context;
+using EasyMeets.Core.DAL.Entities;
+
+namespace EasyMeets.Core.BLL.Services;
+
+public class CalendarEventService : BaseService, ICalendarEventService
+{
+    public CalendarEventService(EasyMeetsCoreContext context, IMapper mapper) : base(context, mapper)
+    {
+    }
+
+    public async Task RemoveCalendarEvents(long calendarId)
+    {
+        var events = _context.CalendarEvents.Where(e => e.CalendarId == calendarId);
+        
+        _context.CalendarEvents.RemoveRange(events);
+        await _context.SaveChangesAsync();
+    }
```

Comment:
```suggestion
    public Task RemoveCalendarEvents(long calendarId)
    {
        var events = _context.CalendarEvents.Where(e => e.CalendarId == calendarId);
        
        _context.CalendarEvents.RemoveRange(events);
        return _context.SaveChangesAsync();
    }
```
More states to the state machine (no, that's bad)

Reply:
Fixed